### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by ops244** [make-all/tuya-local#1399] [Q&A]
hi, I wanted to ask a question, I left a few new devices below, it says full_info new device, what does this mean? will they be added? it's just that there have already been 3 updates and they haven't been added, or am I doing something wrong?

**Comment by make-all** [comment]
A full info tag means there is enough info to work on this.  It will get added eventually, but a lack of time and increasing rate of requests mean that the pipeline is currently around 2 weeks for a pull request and over a month for new device requests in issues.  I do try to clear the PR queue of all PRs that are not waiting on changes, and process some new device requests from issues for each release, but the last release was a hotfix for an issue that came out with the HA 2023.12 release, and the one before it was also rushed before HA 2023.12 for another issue that was detected around a week before its release, so may not have cleared as much from the issues queue as I would have wanted.

**Comment by ops244** [comment]
Thanks for the reply, I will be waiting
